### DISCUSSION TITLE
I m currently trying to set up aws S3 To [Nov 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4438] [Discord Threads]
> 🙋‍♂️ **peehhhh**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Currently we are using the s3 url directly for displaying the image content so if it's not public it won't be visible at the moment and it will show `Access Denied` since it trying to access it publicly  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1038207146363453520)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
